### PR TITLE
E2E: get CoBlocks specs to a green state.

### PR DESCRIPTION
**Pull request by dpasque**
Context: p1680716842686819-slack-CBTN58FTJ

## Proposed Changes

This PR fixes several CoBlocks specs failing in the Gutenberg Simple Edge build.

## Testing Instructions

Ensure the following build configurations are passing:
  - [x] Gutenberg E2E Simple Prod (desktop)
  - [x] Gutenberg E2E Simple Prod (mobile)
  - [x] Gutenberg E2E Simple Edge w/CoBlocks (desktop)
  - [x] Gutenberg E2E Simple Edge w/CoBlocks (mobile)

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by worldomonation** [comment]
Gutenberg E2E Simple Edge w/CoBlocks (desktop) is passing.

image

**Comment by worldomonation** [comment]
CoBlocks related tests passed on Gutenberg E2E Simple Edge w/CoBlocks (mobile). Failures in the screenshot are due to https://github.com/Automattic/wp-calypso/issues/76501 addressed by https://github.com/Automattic/wp-calypso/pull/76503.

image

**Comment by worldomonation** [comment]
Tagging @youbrokesomething because @dpasque is the one that opened the PR, meaning he can't review his own PR.

File: test/e2e/specs/fse/fse__temp-smoke-test.ts [hunk 1]
```diff
@@ -24,12 +24,46 @@ declare const browser: Browser;
...
+		{
+			gutenberg: 'edge',
+			coblocks: 'edge',
+			siteType: 'atomic',
+			variant: 'siteEditor',
+			accountName: 'siteEditorAtomicSiteEdgeUser',
+		},
+	] );
+
+	console.log( accountName );
```
Comment (karenroldan) [inline test/e2e/specs/fse/fse__temp-smoke-test.ts]: **chore:** We can remove this. If we want to continue logging this, we can add a message.

**Review by karenroldan — APPROVED** [review]
LGTM! 🚀 left only 1 chore then we can merge the changes.

File: test/e2e/specs/fse/fse__temp-smoke-test.ts (hunk 1 above)
Comment (worldomonation) [reply test/e2e/specs/fse/fse__temp-smoke-test.ts]: Good eye, didn't see that.

Also @dpasque, wouldn't this be easier if we swap the CoBlocks user out to a block-based theme?

**Pull request merged**